### ISSUE TITLE
Cannot reject image from filmstrip with mouse

### ISSUE THREAD

**Issue by s5k6** [darktable-org/darktable#5852]
**Describe the bug**

When clicking on the little X-symbol on a thumbnail in darkroom's filmstrip, the rating is not updated.  Going back to lightroom will show the old rating as well.  Clickig on one of the five stars seems to work, although I cannot set the rating to not-rejected-but-0-stars using the mouse.

**To Reproduce**

Open image in lightroom.  Open filmstrip.  Click on 3rd star to change rating to 3.  Click in X to change rating to "rejected".  The rating remains at 3.

**Expected behavior**

The rating should be "rejected".

**Platform (please complete the following information):**

    Darktable Version: release-3.1.0-2511-g4226ac17c
    OS: Arch Linux
    OpenCL activated
    Nvidia GeForce GTX 1060

**Comment by phweyland** [comment]
Can see the same issue with 3.1.0+2519~g7592a7bc6

**Comment by s5k6** [comment]
This bug persists with release-3.3.0-845-g9bfedf80d

**Comment by AlicVB** [comment]
From my side I can't reproduce with current master.
There was a bug at some point in may-june (maybe still present in 3.2.1...), but I should have fixed it...
Questions : 
1. Anything special to reproduce ?
2. does it happen with the currently edited image ? with the others ?
3. if you set top filter to "all expect rejected", does the thumbnail disappear ?
4. once you go back to lighttable, is the reject icon in the thumbnail highlighted ?
5. after restarting dt, is the reject icon in the thumbnail highlighted ?

**Comment by s5k6** [comment]
Hello AlicVB

  * Anything special to reproduce ?

    Not that I'd be aware of.  I've made sure to *not* load any of my Lua scripts, and I can still reproduce this.

    It's not a come-and-go thing, it is consistently not working.

    This only affects rejecting an image in the filmstrip.  I can use the mouse to give it, say, 3 stars.  I would not know how to give it zero stars with the mouse.

  * does it happen with the currently edited image ? with the others ?

    With all.

  * if you set top filter to "all expect rejected", does the thumbnail disappear ?

    No, if I set the filter to "all expect rejected", then darktable behaves as if the image was not rejected.  Both, lighttable and drakroom consistently behave as if the image was not rejected.

  * once you go back to lighttable, is the reject icon in the thumbnail highlighted ?

    No (see previous point)

  * after restarting dt, is the reject icon in the thumbnail highlighted ?

    No (see above point)

I assume that the rating of the image never changes.

Unfortunately, I'll be away for another week.  Blease bear with my delayed replies...

**Comment by Nilvus** [comment]
Some ratings updates had been fixed in actual master, since your last comment @s5k6. I just made some tests with your step-by-step provided and I can't reproduce. When setting 3 stars in filmstrip then rejected, image is rejected and no stars set (as expected). Clicking to the X to unreject make stars go back also (as expected too). I so close as this seems to be fixed in actual master, and so on for 3.6 which will be release in about 2 weeks.

If you find another way to reproduce on that 3.6 release, feel free to ask in july to reopen with more precise infos.
